.github/workflows/libjs-test262.yml

```diff
@@ -89,14 +89,22 @@ jobs:
           cmake -GNinja -DCMAKE_C_COMPILER=gcc-11 -DCMAKE_CXX_COMPILER=g++-11 -DSERENITY_SOURCE_DIR=${{ env.SERENITY_SOURCE_DIR }} ..
           ninja libjs-test262-runner test-js
 
+      - name: Get previous results
+        working-directory: libjs-test262
+        run: |
+          cp -f ../libjs-website/test262/data/master.json . || :
+          cp -f ../libjs-website/test262/data/bytecode-master.json . || :
```

Comment:
The ` || :` is needed because in the beginning we won't have the files, we could just add empty files to not have to have this I guess